docker/test/integration/README.md

```diff
@@ -21,7 +21,16 @@ well as other systems which are available in docker, such as Apache NiFi.
 
 * Currently test_https.py does not work due to the upgrade to NiFi 1.7. This will be resolved as
   soon as possible.
-  
+
+## Test environment
+
+The test framework is written in python3 language and uses pip3 package manager to add required packages.
```

Comment:
```suggestion
The test framework is written in Python 3 and uses pip3 to add required packages.
```

- "written in" implies a programming language, so removed "language".
- Changed "python3" to "Python 3" as "python3" is a command used by many (but not all) distros to launch the interpreter, but I think the name and version of the language should be written as "Python 3".
- removed "package manager" because is sounds strange for me without an article ("the"), and is definitely redundant.
- Noting that "pip3" is also just a command to invoke "pip", the package manager, but either fits here.

English is not my first language so I may be wrong in some of the above.